src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
-}
+  let sum = 0;
+
+  return function plus(elem) {
+    if (elem === undefined) {
+      const displayResult = sum;
+
+      sum = 0; // sum reset
```

Comment:
```suggestion
      sum = 0;
```
it's understandable)

Reply:
:+1: :smile: